dbux-graph-host/src/components/controllers/HighlightManager.js

```diff
@@ -1,11 +1,30 @@
+import NanoEvents from 'nanoevents';
 import HostComponentEndpoint from 'dbux-graph-host/src/componentLib/HostComponentEndpoint';
 
 export default class HighlightManager extends HostComponentEndpoint {
   init() {
     this.state.highlightAmount = 0;
+    this.allHighlighter = new Set();
+    this._emitter = new NanoEvents();
   }
 
-  highlighterUpdated(newState) {
+  registHighlight(highlighter, newState) {
+    if (newState === 1) this.allHighlighter.add(highlighter);
```

Comment:
I'd prefer curly braces `{ }` for all `if`s - how do you think about that?

Reply:
Also, would make sense to:
1. change `newState` to `enabled`
2. make it boolean
3. rename `registHighlight` to `setHighlighterEnabled`